app/views/super_admin/organisations/index.html.erb

```diff
@@ -29,34 +29,34 @@
 <table class="govuk-table">
   <thead class="govuk-table__head">
     <tr class="govuk-table__row">
-      <th class="govuk-table__header govuk-!-width-one-half"><%= sort_link "name" %></th>
-      <th class="govuk-table__header"><%= sort_link "created_at", "Created on" %></th>
-      <th class="govuk-table__header govuk-table__header"><%= sort_link "mous.created_at", "MOU Signed" %></th>
-      <th class="govuk-table__header govuk-table__header--numeric"><%= sort_link "locations_count", "Locations" %></th>
-      <th class="govuk-table__header govuk-table__header--numeric"><%= sort_link "ips_count", "IPs" %></th>
-      <th class="govuk-table__header"><%= sort_link "certificates_count", "Uses EAP-TLS" %></th>
+      <th class="govuk-table__header govuk-!-width-one-half" scope="row"><%= sort_link "name" %></th>
+      <th class="govuk-table__header"><%= sort_link "created_at", "Created on" scope="row" %></th>
```

Comment:
There's a comma missing after `"Created on"`

Reply:
Thank you - comma added